### DISCUSSION TITLE
is it possible to convert gpt-j to gguf?

### DISCUSSION THREAD

**Discussion by silvacarl2** [ggml-org/llama.cpp#10871] [Q&A]
is it possible to convert gpt-j to gguf?

**Comment by danbev** [comment]
This should be possible using [convert_hf_to_gguf.py](https://github.com/ggerganov/llama.cpp/blob/master/convert_hf_to_gguf.py), since it is one of the [model architectures](https://github.com/ggerganov/llama.cpp/blob/0e70ba686e6c717a0aa41d88284e2a392c2bd0cd/gguf-py/gguf/constants.py#L217) supported.

**Comment by silvacarl2** [comment]
thx you, checking that out.
